tests/test_aggr_regressions.py

```diff
@@ -93,6 +93,20 @@ class X(dj.Lookup):
     """
     contents = zip(range(10))
 
+@schema
+class TableA(dj.Manual):
+    definition = """
+    table_a: int
+    """
+
+
+@schema
+class TableB(dj.Manual):
+    definition = """
+    -> TableA
+    table_b: int
+    """
+
```

Comment:
@jverswijver This seems like a pretty simple/generic pipeline setup. Nice job in identifying one! Since it is so simple we can probably just reuse one of our other existing ones. How about?
- `tests/schema.py:UberTrash`
- `tests/schema.py:UnterTrash`